libs/clients/assets/src/lib/properties-api.provider.ts

```diff
@@ -0,0 +1,48 @@
+import { Provider } from '@nestjs/common/interfaces/modules/provider.interface'
```

Comment:
Name of file not following naming conventions.

ref: https://docs.devland.is/technical-overview/adr/0009-naming-files-and-directories#decision-outcome

Reply:
This is a bit annoying since it goes against the NX templates and NestJS conventions.

Our repo is now full of `something.module|resolver|controller|model|dto.ts` files... :/

Reply:
Yeah. I saw that you changed this to Pascal case, which is incorrect. Pascal case should only be for React components.

The filename should follow camelcase.
From docs: "If that default export is a React Component, or a class, then the file name should be in PascalCase. Otherwise, the filename should be in camelCase."

Your file is not a React component, nor does it contain a default export... so the filename should be in camelCase.

Reply:
Which would agree with our repo's files:
```
Our repo is already full of something.module|resolver|controller|service|model|dto.ts files
```

Reply:
Hm, I think the decision may need a review.

First, I feel like the ecosystem is generally moving away from default exports and always use named exports, which are arguably more consistent, are nicer to use in tunnel files and IntelliSense. So we should maybe document that and define naming conventions for modules which contain a "primary export" (as opposed to a collection of similar exports like in a `utils.ts` module).

My intention when originally reviewing that decision was that the filename should be written exactly the same as the "primary export". I feel like "React Components and classes" is not an exhausted list of pascal cased filenames. IMHO it's an unnecessary mental load to think about this arbitrary classification of the primary export when naming the file. I.e. why is the Button file pascal-cased and not files exporting SomeEnum, SomeConstant, SomeType or SomeInterface?

Also, most of the repo files I listed have a "primary export" of a class. So according to the naming convention they should be named AuthService.ts rather than auth.service.ts or Auth.service.ts, right? Meaning we should get rid of all dots from filenames, except some postfixes like `.spec.ts`.